shared/util.go

```diff
@@ -94,6 +127,15 @@ var (
 	lck = LoggerCtxKey{}
 )
 
+// NewLoggerMux creates a multiplexing Logger that writes all log operations to
+// all delegates.
+func NewLoggerMux(delegates []Logger) Logger {
+	if delegates == nil || len(delegates) == 0 {
```

Comment:
Nit: `len(delegates) == 0` would suffice, since `delegates` has a concrete slice type (you can't do `len(nil)`, but you can `len([]int(nil))`)

Reply:
Done.